### PR TITLE
device doc update

### PR DESCRIPTION
**Pull request by thijstriemstra**

### CODE REVIEW

**Comment by thijstriemstra** [comment]
When you mouse-over the devices in the diagram on https://luma-oled.readthedocs.io/en/latest/api-documentation.html#module-luma.oled you'll notice the 'encapsulated' part is not needed.

File: doc/conf.py [hunk 1]
```diff
@@ -64,7 +66,8 @@
 master_doc = 'index'
 
 # General information about the project.
-project = 'Luma.OLED: Display drivers for SSD1306 / SSD1322 / SSD1325 / SSD1331 / SSD1351 / SH1106'
+project = 'Luma.OLED: Display drivers for {0}'.format(
+    '/ '.join(supported_devices).upper())
```
Comment (rm-hull) [inline doc/conf.py]: Maybe just use a comma like in the other PR

Comment (thijstriemstra) [reply doc/conf.py]: ok cool

Comment (thijstriemstra) [reply doc/conf.py]: I also hope it solves the missing device on https://luma-oled.readthedocs.io/en/latest/api-documentation.html#module-luma.oled although I'm not sure where that line's coming from..

```
OLED display driver for SSD1306, SSD1322, SSD1325, SSD1331 and SH1106 devices.
```

**Comment by thijstriemstra** [comment]
@rm-hull the SSD1351 is also missing in the github description.

**Comment by rm-hull** [comment]
^ fixed

**Pull request merged**